.appveyor.yml

```diff
@@ -2,17 +2,27 @@ version: 0.8.{build}
 environment:
   matrix:
     # array of all environments used to test builds
+    - GENERATOR: NMake Makefiles
+      CFG: Debug
+      VS_VERSION: 12.0
     - GENERATOR: Visual Studio 14 2015
+      VS_VERSION: 14.0
       CFG: Debug
     - GENERATOR: Visual Studio 12 2013
+      VS_VERSION: 12.0
       CFG: Debug
     - GENERATOR: Visual Studio 14 2015 Win64
       CFG: Debug
+      VS_VERSION: 14.0
     - GENERATOR: Visual Studio 12 2013 Win64
       CFG: Debug
+      VS_VERSION: 12.0
+install:
+  - gem install asciidoctor
```

Comment:
How much time does this add to the build? Probably not much, but it's worth monitoring.